schema/bom-1.6.schema.json

```diff
@@ -2320,16 +2318,36 @@
       ]
     },
     "version": {
-      "description": "A single version of a component or service.",
+      "description": "A single disjunctive version identifier, for a component or service.",
       "type": "string",
       "minLength": 1,
-      "maxLength": 1024
+      "maxLength": 1024,
+      "examples": [
+        "9.0.14",
+        "v1.33.7",
+        "7.0.0-M1",
+        "2.0pre1",
+        "1.0.0-beta1",
+        "0.8.15"
+      ]
     },
-    "range": {
+    "version-range": {
```

Comment:
If I'm not mistaken, we use lower camel case for multi-word definitions. I'd like to be consistent. Therefore I think it should be `versionRange`.

Reply:
you are right. fixed that.